test/assets.test.ts

```diff
@@ -7,6 +7,7 @@ import { should } from 'chai';
 import * as path from 'path';
 import * as protocol from '../src/omnisharp/protocol';
 import { AssetGenerator } from '../src/assets';
+import * as json from 'comment-json';
```

Comment:
Please don't use 'comment-json'. [jsonc-parser](https://www.npmjs.com/package/jsonc-parser) is provided by VS Code to parse JSON files with comments and we already list it as a dependency. Please use that rather than adding another dependency for the same functionality.